src/zos-spawn.cc

```diff
@@ -15,6 +15,8 @@
 #define _POSIX_SOURCE
 #include <unistd.h>
 
+#include "zos-io.h"
```

Comment:
Should move this above include of unistd.h.  That will make sure zos-io.h is self contained (i.e. not missing any includes/decls).  This should be common practice in the source files.

Reply:
I think that would apply if src/zos-io.cc was being changed, in which case include "zos-io.h" would be very the first include - according to https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes.

Reply:
Yes, which actually means this file should have spawn.h as the first include.

Reply:
It already does.